### PR TITLE
[FLINK-6701] Activate strict checkstyle for flink-yarn

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

File: flink-yarn/src/main/java/org/apache/flink/yarn/YarnResourceManager.java [hunk 1]
```diff
@@ -37,75 +37,74 @@
...
 import java.util.concurrent.TimeUnit;
 
+import scala.concurrent.duration.FiniteDuration;
+
 /**
  * The yarn implementation of the resource manager. Used when the system is started
  * via the resource framework YARN.
  */
 public class YarnResourceManager extends ResourceManager<ResourceID> implements AMRMClientAsync.CallbackHandler {
-	protected final Logger LOG = LoggerFactory.getLogger(getClass());
```
Comment (greghogan) [inline flink-yarn/src/main/java/org/apache/flink/yarn/YarnResourceManager.java]: Is it intended that the logger be inherited from `RpcEndpoint`?

Comment (zentol) [inline flink-yarn/src/main/java/org/apache/flink/yarn/YarnResourceManager.java]: Yes. They both call `getClass()` and thus should look the same from the outside.

**Comment by zentol** [comment]
@greghogan I've rebased the branch and addressed your comments.

**Comment by greghogan** [comment]
+1

**Pull request closed without merging**